ci/workflows/main-cron.yml

```diff
@@ -10,10 +10,10 @@ anchors:
         run: rw-build-env
         config: ci/docker-compose.yml
         mount-buildkite-agent: true
+        propagate-environment: true
```

Comment:
So that envs like `BUILDKITE_PARALLEL_JOB` can be passthrough into the containers and read by `sqllogictest-rs`.